node/workload/Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM alpine:3.8
+FROM alpine:3.11
```

Comment:
Last `alpine` release that still has `python` (python2) package. Our node STs are still using python2.